### ISSUE TITLE
Category and Tag labels on posts use h4s

### ISSUE THREAD

**Issue by trutteriastate** [iastate/iastate22-wordpress#78]
The **Categories:** and **Tags:**  labels on posts use H4 level headings, which may result in unstructured heading warnings and accessibility issues.

Suggest they are simply styled rather than use a heading label.

https://www.w3.org/WAI/tutorials/page-structure/headings/#:~:text=heading%20after%20navigation-,Heading%20ranks,it%20closes%20the%20previous%20section.

_Skipping heading ranks can be confusing and should be avoided where possible: Make sure that a h2 is not followed directly by an h4, for example. It is ok to skip ranks when closing subsections, for instance, a h2 beginning a new section, can follow an h4 as it closes the previous section._

**Comment by kwickham** [comment]
Will need to consult with Theme group since it's part of the base template.

https://www.theme.iastate.edu/iastate22/fractal/components/detail/article.html
